### DISCUSSION TITLE
Welcome to wikiapi Discussions! 歡迎來到 wikiapi 討論區！

### DISCUSSION THREAD

**Discussion by kanasimi** [kanasimi/wikiapi#59] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 歡迎!
  我們使用 GitHub Discussions 作為社區成員聯系的地方。我們希望您：
  * 提出想解決的問題。
  * 分享想法或建議。
  * 與其他社區成員交流。
  * 歡迎他人，並保持開放的心態。請記住，這是我們共同建立的社區💪。

  如需開始，在下方用評論介紹您自己，並告訴我們您對什麼感興趣或者想要做哪些有趣的事。

**Comment by hugolpz** [comment]
Hello Kanasimi, Happy to hear from you. I just returned to coding my bot few hours ago, and it is doing its [first mass edits](https://commons.wikimedia.org/wiki/Special:Contributions/Dragons_Bot) on Cantonese and Chinese recordings. I will have questions emerging from that initiative, so this Github discussion forum is welcome. Hope you are going well !

**Comment by kanasimi** [comment]
Thank you for your participation!
I'm sorry this is a welcome text. I don't know if you can start a new discussion?

**Reply by hugolpz** [reply]
From what I see it work as a single page form. We can post a comment, or "reply" under one.
I currently don't see how to create additional discussion pages. I only see how to comment / reply after you.
Note: 2am here, need to sleep. 😴

**Reply by kanasimi** [reply]
You may start a new discussion [here](/kanasimi/wikiapi/discussions/new).

**Reply by hugolpz** [reply]
Oh ! this here isn't the root page, nice. I will dig this tomorrow. (I will also remove few out of topic messages I posted today.)
